### ISSUE TITLE
Formatting issue in output during errors

### ISSUE THREAD

**Issue by therohk** [gtg922r/obsidian-numerals#66]
There is a bug in the formatting of an output block when math.eval results in an error.

The code linked [here](https://github.com/gtg922r/obsidian-numerals/blob/master/src/numeralsUtilities.ts#L551) references a css class that is not present in styles.css
This causes the error line to be misaligned in all layout modes.

Here is small fix that resolves this issue: https://github.com/gtg922r/obsidian-numerals/pull/65

Thanks for building this indispensable plugin!

**Comment by gtg922r** [comment]
@therohk Good catch! thanks for the PR. I'm not seeing a difference in the rendered result though. Can you share a screenshot of the difference by chance?

**Comment by gtg922r** [comment]
Whoops, that was my bad. Just needed to refresh the block. I see it now

**Comment by gtg922r** [comment]
Fixed now. Thanks again
